.pre-commit-config.yaml

```diff
@@ -24,7 +24,7 @@ repos:
     rev: 6.0.0
     hooks:
       - id: flake8
-        args: ["--ignore=E203,W503,W605", "--max-line-length=125"]
+        args: ["--ignore=E203,W503,W605", "--max-line-length=160", "--extend-exclude=src/aws/osml/formats"]
```

Comment:
Would we like to standardize on 160 in other packages as the max line length?

Reply:
The only reason I did this is that there are some long regular expression strings in the XML parsing code. They look really ugly if they get broken up. 

The black formatter still sets things to 125 which seems like a generally good practice. The only reason I set this here is because flake8 seems to be ignoring that excludes option so it was still complaining about the XML format files.